glib/PointerWrapper.cs

```diff
@@ -23,35 +23,26 @@
 
 namespace GLib
 {
-	class PointerWrapper : IDisposable
+	sealed class PointerWrapper : IDisposable
 	{
-		internal ToggleRef tref;
-		internal IntPtr handle;
-
-		static Action<IntPtr> ObjectCreated;
-		static Action<IntPtr> ObjectDestroyed;
```

Comment:
Were these two just little helpers for allowing people to track creation/destruction using external tools and reflection? If so can we leave them in place? They don't do any harm as such :)

Reply:
Sure